### PR TITLE
chore: update circle config

### PR DESCRIPTION
**Pull request by beerecca**
@spirosikmd 🤞

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          requires:
+            - test_and_npm_release
+  prod:
+    jobs:
+      - test_and_npm_release:
+          filters:
+            branches:
+              only: master
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*/
```
Comment (spirosikmd) [inline .circleci/config.yml]: As far as I can see the previous condition to run the production build was `tag: /v[0-9]+(\.[0-9]+)*/`. Now we have both the `branch: master` and the `tag: /v[0-9]+(\.[0-9]+)*/`. Should we only keep the tag?

Comment (beerecca) [reply .circleci/config.yml]: Sure

**Comment by beerecca** [comment]
@spirosikmd ahh nope, think we still want that there! 
screen shot 2018-08-07 at 15 58 10

**Comment by beerecca** [comment]
@spirosikmd actually we might be able to not have it, but that tags filter didn't seem to work at all!

**Pull request merged**